### PR TITLE
Customize react/jsx-props-no-spreading rule

### PR DESCRIPTION
**Pull request by dozortsev**
## ❓ Context

Rule: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md

See comments in code.

## 🚀 Changes

Customize the rule.

## ✅ Checklist

- [x] The package version is bumped according to [semver](https://semver.org) in `package.json`, `package-lock.json`, and `CHANGELOG.md`

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -10,6 +10,13 @@ module.exports = {
   plugins: ['fp'],
   extends: ['airbnb', 'plugin:prettier/recommended', 'prettier/react'],
   rules: {
+    'react/jsx-props-no-spreading': [
+      2,
+      {
+        explicitSpread: 'ignore',
```
Comment (dozortsev) [inline .eslintrc.js:16]: We would like to allow explicit spreading:
```js
<Switch {...{ onClick, label }} />
```

File: .eslintrc.js [hunk 2]
```diff
@@ -10,6 +10,13 @@ module.exports = {
...
+        custom: 'ignore',
```
Comment (dozortsev) [inline .eslintrc.js:17]: and props spreading for custom components:

```js
<Switch {...props} />
```

**Review by sbrandwoo — APPROVED** [review]
:+1:

**Comment by sbrandwoo** [comment]
Do you know why yarn.lock changed?

**Comment by dozortsev** [comment]
> Do you know why yarn.lock changed?

@sbrandwoo not exactly sure, but this that last time someone forget to do `yarn install`

**Comment by twrdjpalmer** [comment]
@dozortsev did you do `yarn install` or `yarn install --frozen-lock-file`? Would recommend the latter when installing deps after cloning the project, so you don't inadvertently bump versions :)

**Comment by dozortsev** [comment]
@twrdjpalmer okey, done

what about the rule change? could you please approve PR? 🙂

**Review by twrdjpalmer — APPROVED** [review]

**Pull request merged**